SportsHub/SportsHub.Api/Controllers/ArticleController.cs

```diff
@@ -82,6 +83,28 @@ public async Task<IActionResult> CreateArticleAsync([FromBody] CreateArticleDTO
             return BadRequest(ValidationMessages.UnableToCreateArticle);
         }
         
+        [Authorize(Roles = "Admin")]
```

Comment:
I think we should add it to the constant or use Enum value

Reply:
This is from `main` and not done by my, should I still implement it?